integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml

```diff
@@ -55,71 +55,89 @@ spec:
           - name: revision
             value: main
           - name: pathInRepo
-            value: common/tasks/rosa/hosted-cp/rosa-hcp-metadata/rosa-hcp-metadata.yaml
-    - name: test-metadata
+            value: common/tasks/test-metadata/0.1/test-metadata.yaml
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: test-name
+          value: $(context.pipelineRun.name)
+    - name: provision-kind-cluster
+      runAfter:
+        - test-metadata
+      when:
+        - input: "$(tasks.test-metadata.results.pull-request-author)"
+          operator: notin
+          values: ["red-hat-konflux[bot]"]
       taskRef:
         resolver: git
         params:
           - name: url
-            value: https://github.com/konflux-ci/tekton-integration-catalog.git
+            value: https://github.com/konflux-ci/tekton-integration-catalog
           - name: revision
             value: main
           - name: pathInRepo
-            value: common/tasks/test-metadata/0.1/test-metadata.yaml
+            value: tasks/mapt-oci/kind-aws-spot/provision/0.1/kind-aws-provision.yaml
       params:
-        - name: SNAPSHOT
-          value: $(params.SNAPSHOT)
-        - name: test-name
+        - name: secret-aws-credentials
+          value: mapt-kind-secret
+        - name: id
+          value: $(context.pipelineRun.name)
+        - name: tags
+          value: env=konflux,user=release-service-catalog
+        - name: debug
+          value: 'false'
+        - name: ownerKind
+          value: PipelineRun
+        - name: ownerName
           value: $(context.pipelineRun.name)
-    - name: provision-rosa
+        - name: ownerUid
+          value: $(context.pipelineRun.uid)
+    - name: deploy-konflux
       when:
         - input: "$(tasks.test-metadata.results.pull-request-author)"
           operator: notin
           values: ["red-hat-konflux[bot]"]
       runAfter:
-        - rosa-hcp-metadata
-        - test-metadata
+        - provision-kind-cluster
       taskRef:
         resolver: git
         params:
           - name: url
-            value: https://github.com/konflux-ci/tekton-integration-catalog.git
+            value: https://github.com/psturc/tekton-integration-catalog
```

Comment:
This doesn't look right. If you're testing something, then I'd recommend changing this PR to Draft.

Reply:
ah you're right, I missed that